index.php

```diff
@@ -30,6 +33,32 @@ function number_project_tasks(array $tasks_list, string $project_name) {
     return $task_counter;
 }
 
+function hours_left_deadline($date_completion) {
+    if ( is_null($date_completion) ) return null;
```

Comment:
Можем обернуть `return ...;` в фигурные скобки?
Принято оборачивать всегда, даже если внутри один оператор.
Кстати, по PSR, открывающая и закрывающая скобки не могут быть на одной строке.
```suggestion
    if (is_null($date_completion)) {
        return null;
    }
```